HCIDOCS-137: Update static deploy steps to use virtual media (#93223)
openshift/openshift-docs: modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

* Configure a `PTR` DNS record with a valid hostname for each node with a static IP address.
* Install the NMState CLI (`nmstate`).

[IMPORTANT]
====

> If you use a provisioning network, configure it by using Ironic's `dnsmasq`. To do a fully static deployment, you must use virtual media.

Author: Thanks so much for the feedback! Yes, Ironic is a product name. I've made the suggested change.

> If you're using a provisioning network, it must be configured by Ironic's `dnsmasq`. In order to do a fully static deployment, you must use virtual media.

Reviewer: ```suggestion
If you are using a provisioning network, it must be configured by Ironic's `dnsmasq`. To perform a fully static deployment, you must use virtual media.
```
https://redhat-documentation.github.io/supplementary-style-guide/#conversational-style - suggestion to avoid contractions.

Reviewer 2: This suggestion is correct, however, the style guide encourages us to use simple present tense.

> If you're using a provisioning network, it must be configured by Ironic's `dnsmasq`. In order to do a fully static deployment, you must use virtual media.

Reviewer 2: ```suggestion
If you use a provisioning network, configure it by using Ironic's `dnsmasq`. To do a fully static deployment, you must use virtual media.
```

Author: Updated as suggested.

> If you're using a provisioning network, it needs to be configured by Ironic's `dnsmasq`, which requires virtual media to configure a fully static deployment.

Reviewer 3: The "which" between the two statements suggests that they're referring to the same thing. They're two completely separate things though - either you use a provisioning network with dnsmasq, or you are doing a fully static deployment with virtual media. I would suggest something like:

If you're using a provisioning network, it needs to be configured by Ironic's `dnsmasq`. In order to do a fully static deployment, you must use virtual media.
